.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  // Place your settings in this file to overwrite default and user settings.
```

Comment:
comments are not supported in json?

Reply:
it's weird that VSCode put these in mine by default. I can remove them..